### PR TITLE
Fix for HELM so it can't impact other nodes/pods

### PR DESCRIPTION
**Pull request by MattiasGees**
HELM has a bug that makes kubelet use a lot of memory https://github.com/kubernetes/helm/issues/3121
To mitigate this until it is solved, we will run HELM on a seperate node so it can't affect other nodes/pods

Tested this on our test cluster both with true and false

### CODE REVIEW

File: cluster/main.tf [hunk 1]
```diff
@@ -148,6 +168,7 @@ data template_file "kops_full_cluster-spec_file" {
     content_cluster      = "${data.template_file.cluster-spec.rendered}"
     content_master_group = "${join("\n",data.template_file.master-instancegroup-spec.*.rendered)}"
     content_worker_group = "${data.template_file.worker-instancegroup-spec.rendered}"
+    content_helm_node    = "${var.helm_node == true ? data.template_file.helm-instancegroup-spec.rendered : ""}"
```
Comment (iuriaranda) [inline cluster/main.tf]: shouldn't you put `true` with `"` here?

File: cluster/variables.tf [hunk 2]
```diff
@@ -64,3 +64,8 @@ variable "environment" {
   description = "Environment where this node belongs to, will be the third part of the node name. Defaults to ''"
   default     = ""
 }
+
+variable "helm_node" {
+  description = "Do we want a seperate node to deploy helm"
+  default     = false
```
Comment (iuriaranda) [inline cluster/variables.tf]: here too, according to the readme

File: cluster/main.tf (hunk 1 above)
Comment (MattiasGees) [reply cluster/main.tf]: No that is not needed.

File: cluster/variables.tf (hunk 2 above)
Comment (MattiasGees) [reply cluster/variables.tf]: No that is not needed.

**Pull request merged**